src/components/screens/ScoreSetEditor.vue

```diff
@@ -1243,6 +1267,7 @@ export default {
         }
         this.referenceGenome = this.item.referenceGenome
         this.assembly = this.item.assembly
+        this.geneName = this.item.geneName
```

Comment:
don't believe there is a geneName on this.item. as the dropdown/autocomplete part we're using is only used during scoreset creation and this is only invoked when fetching a completed scoreset, probably best to skip this.

Reply:
Yes, you're correct.